src/30_Advanced/Embedding_amp-video_in_amp-carousel_and_amp-accordion.html

```diff
@@ -0,0 +1,235 @@
+  <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
+  <script async custom-element="amp-vimeo" src="https://cdn.ampproject.org/v0/amp-vimeo-0.1.js"></script>
+  <script async custom-element="amp-dailymotion" src="https://cdn.ampproject.org/v0/amp-dailymotion-0.1.js"></script>
+  <script async custom-element="amp-twitter" src="https://cdn.ampproject.org/v0/amp-twitter-0.1.js"></script>
+  <script async custom-element="amp-facebook" src="https://cdn.ampproject.org/v0/amp-facebook-0.1.js"></script>
+  <script async custom-element="amp-brightcove" src="https://cdn.ampproject.org/v0/amp-brightcove-0.1.js"></script>  
+  <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+  section[expanded] span.expanded {
+    display: block;
+  }
+  section[expanded] span.collapsed {
+    display: none;
+  }
+  section:not([expanded]) span.expanded {
+    display: none;
+  }
+  section:not([expanded]) span.collapsed {
+    display: block;
+  }
+  .blue-box, .red-box, .green-box {
+      width: 100%;
+      height: 280px;
+    }
+    .blue-box {
+      background: blue;
+    }
+    .green-box {
+      background: green;
+    }
+    .red-box {
+      background: red;
+    }
+</style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### amp-video embedded in an amp-accordion component -->
+  <!--
+    Each of the `amp-accordion` component’s immediate children is considered a section in the accordion. For more info about `amp-accordion` take a look [here](https://ampbyexample.com/components/amp-accordion/).
+  -->
+  <amp-accordion>
+    <section>
+      <h4>amp-video - Click to expand</h4>
+      <figure>
+        <amp-video width=480 height=270 src="video/tokyo.mp4"
+               poster="/img/tokyo.jpg"
+               layout="responsive"
+               controls>
+        <div fallback>
+        <p>Your browser doesn't support HTML5 video.</p>
+        </div>
+        <source type="video/mp4" src="video/tokyo.mp4">
```

Comment:
the video doesn't work. Change the src to "/video/tokyo.mp4" and "/video/tokyo.webm"